src/condor_credd/condor_credmon_oauth/examples/config/condor/40-oauth-credmon.conf

```diff
@@ -57,6 +57,16 @@ use feature : OAUTH
 # or as bare strings (LOCAL_CREDMON_TOKEN_USE_JSON = false)
 # LOCAL_CREDMON_TOKEN_USE_JSON = false
 
+# Audience (aud) that should be set in the token.
+# Audience is required for scitokens:2.0 tokens, which are the most common.
+# The audience is a shared value between the token issuer and the service verifying the token.
+# It is a space separated list, will be converted to a json list in the token
+# LOCAL_CREDMON_TOKEN_AUDIENCE = https://example.com https://anotherserver.edu
```

Comment:
If I'm reading the code right, a blank `aud` is no good? If so, this should be under `MANDATORY for local credmon`.

Reply:
`aud` is optional in scitokens:1.0.  But mandatory for 2.0.

Reply:
Let's stick this under `MANDATORY` then since `scitokens:2.0` will be the default `ver`.

Reply:
It's only mandatory if you are using the local issuer.  Similar to how "mandatory" the scitokens private key is.